sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -652,6 +640,23 @@ private Response<RecognizePiiEntitiesResultCollection> RecognizePiiEntitiesBatch
             }
         }
 
+        private static PiiTaskParameters PiiEntitiesParameters(RecognizePiiEntitiesOptions options)
+        {
+            PiiTaskParameters parameters = new()
+            {
+                LoggingOptOut = options.DisableServiceLogs,
+                ModelVersion = options.ModelVersion,
+                Domain = options.DomainFilter.GetString() ?? (PiiDomain?)null,
+                StringIndexType = Constants.DefaultStringIndexType
+            };
+
+            if (options.CategoriesFilter.Count > 0)
+            {
+                parameters.PiiCategories = options.CategoriesFilter;
```

Comment:
just by touching `PiiCategories`, even if it is to assign an empty list, codegen will add it to the content of the request (empty value though), but we don't want that. 
We had this problem before and did this (see [here](https://github.com/Azure/azure-sdk-for-net/blob/feature/text-analytics-language-api/sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.Legacy.cs#L486) in the Legacy code for PiiActions). There might be a better solution, but putting this one out for now.